man/resolvectl.xml

```diff
@@ -367,7 +369,9 @@
 
     </variablelist>
 
-    <para>See <citerefentry><refentrytitle>resolvconf</refentrytitle><manvolnum>8</manvolnum></citerefentry> for details on this command line options.</para>
+    <para>See
+    <citerefentry project="ubuntu"><refentrytitle>resolvconf</refentrytitle><manvolnum>8</manvolnum></citerefentry>
```

Comment:
i guess the ubuntu link works too, but i wonder if the debian link for this might be more appropriate, given it's more "upstream" for this? dunno?

it would be:

https://manpages.debian.org/buster/resolvconf/resolvconf.8.en.html

Reply:
In that case, I'd probably replace `buster` with either `stable` or `unstable`, just so the link is not tied to a specific release and remains up-to-date

Reply:
Yeah, that's better. Updated.